standard-covoiturage_openapi.yaml

```
@@ -0,0 +1,506 @@
+openapi: 3.0.1
+info:
+  title: ""
+  description: ""
+  version: 0.0.1
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+tags:
+  - name: Search
+    description: API routes to search for journeys. These routes can be left opened to public.
+
+#
+# API Routes
+#
+paths:
+  /driver_journeys:
+    get:
+      tags:
+      - Search
+      summary: Search for matching punctual planned outward driver journeys.
```

Could we clarify this summary? What does it mean `punctual`, `planned` and `outward`?  I propose we remove those terms from the summary and description.

```suggestion
      summary: Search for available driver journeys matching the provided criteria.
```